connectors/es/sink.py

```diff
@@ -884,7 +884,7 @@ def _extractor_task_running(self):
     async def cancel(self):
         if self._sink_task_running():
             self._logger.info(
-                f"Cancling the Sink task: {self._sink_task.name}"  # pyright: ignore
+                f"Canceling the Sink task: {self._sink_task.get_name()}"  # pyright: ignore
```

Comment:
Just curious, what was the purpose of `# pyright: ignore`? Maybe just to retain the line breaks? Because removing it doesn't anger the linter.

Reply:
huh does it not for you? My local linter is messed up, but I'd added these originally because it was telling me that `_sink_task` might be `None`. I know that `if self._sink_task_running()` already checked that, but the linter didn't seem to.

I'll try removing them and see if CI will allow it, though.

Reply:
It seemed fine for my linter, it just warned me that it would autoformat that section to be one line. But yes let's let CI decide.